app/views/applications/create_application_success.scala.html

```diff
@@ -68,10 +68,15 @@ <h1 class="heading-large">Application added</h1>
         }}
     </tr>
     @if(totpSecret.isDefined) {
-    <tr>
-        <th>TOTP secret</th>
-        <td>@totpSecret.get.production</td>
-    </tr>
+        <tr>
+            <th>TOTP secret</th>
+            <td>@totpSecret.get.production</td>
+        </tr>
+    } else {
+        <tr>
+            <th>Client secret</th>
+            <td>@clientSecret.get</td>
```

Comment:
Not really a PR comment, but for my understanding - "Why does TOTP have separate production and sandbox secrets but the client secret doesn't?"